README.rst

```diff
@@ -22,6 +25,16 @@ Until May 2016, Certbot was named simply ``letsencrypt`` or ``letsencrypt-auto``
 depending on install method. Instructions on the Internet, and some pieces of the
 software, may still refer to this older name.
 
+Certbot is packaged for many common operating systems and web servers. Check whether
+``certbot`` (or ``letsencrypt``) is packaged for your web server's OS by visiting
+certbot.eff.org_, where you will also find the correct installation instructions for
+your system.
+
+.. Note:: Unless you have very specific requirements, we kindly suggest that you use the Certbot packages provided by your package manager (see certbot.eff.org_). If such packages are not available, we recommend using ``certbot-auto``, which automates the process of installing Certbot on your system.
+
+.. _certbot.eff.org: https://certbot.eff.org
```

Comment:
if you're going to include this then you need to edit the one on line 53, otherwise they're duplicates